webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/db/DB.scala

```diff
@@ -110,7 +111,7 @@ final object DB {
 
   def insertReports(reports: List[com.normation.rudder.domain.reports.Reports]): ConnectionIO[Int] = {
     val dbreports = reports.map { r =>
-      DB.Reports[Unit]((), r.executionDate, r.nodeId.value, r.directiveId.value, r.ruleId.value, r.serial
```

Comment:
We want to insert the old directive id or the id with revision ?

Reply:
And rule is still the id and not the RId (but they should be the same between the Rule and the Directive, i think)

Reply:
Yes, for now you can't version rule, it's the next step